app/src/main/res/layout/fragment_favorites.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Is there a reason we use a FrameLayout as a parent for this layout?

Generally you don't need to wrap a layout in a FrameLayout